app/src/main/java/de/westnordost/streetcomplete/quests/summit_register/AddSummitRegister.kt

```diff
@@ -45,21 +45,27 @@ class AddSummitRegister : OsmElementQuestType<Boolean> {
         val peaks = mapData.nodes.filter { filter.matches(it) }
         if (peaks.isEmpty()) return emptyList()
 
-        val hikingRoutes = mapData.relations
-            .filter { it.tags["route"] == "hiking" }
-            .mapNotNull { mapData.getRelationGeometry(it.id) as? ElementPolylinesGeometry }
-        if (hikingRoutes.isEmpty()) return emptyList()
+        val hikingPathsAndRoutes = mapData.ways.filter { hikingPathsFilter.matches(it) }
+            .mapNotNull { mapData.getWayGeometry(it.id) as? ElementPolylinesGeometry } +
+            mapData.relations.filter {  it.tags["route"] == "hiking" }
+                .mapNotNull { mapData.getRelationGeometry(it.id) as? ElementPolylinesGeometry }
 
         // yes, this is very inefficient, however, peaks are very rare
         return peaks.filter { peak ->
-            hikingRoutes.any { hikingRoute ->
-                hikingRoute.polylines.any { ways ->
+            hikingPathsAndRoutes.any { hikingPath ->
```

Comment:
> I removed always asking on nodes with summit:cross because the only comment on this seemed to not be in favor of using it:
>
>> I am pretty sure that some crosses are on summits inaccessible by regular hikers

(by @Helium314 in https://github.com/streetcomplete/StreetComplete/pull/4075#discussion_r892125037 and @matkoniecz in https://github.com/streetcomplete/StreetComplete/pull/4075#discussion_r888950833)

Is it an issue though? If someone searches for quests in an inaccessible area, they either know how to walk or climb there or they will ignore the quest (or deliberately answer from a distance…).

I think that asking for all nodes with `summit:cross = yes` makes sense and will make the quest more useful again. Also, if `summit:register` was already tagged, resurveying should always be enabled IMHO, regardless whether a way/route is nearby.

Reply:
I also think it makes sense, but @matkoniecz didn't like it.
I don't need to deal with people opening you-want-the-users-to-get-in-dangerous-situations issues, but I understand the desire to avoid them.

Reply:
I am not blocking it strongly - if you are sure that it is a good idea (especially if checking nearby sac_scale paths is not going to work or will exclude too many) then add it.

Reply:
reading vertically through this, Flo's point makes sense to me